examples/app-with-rbac/src/pages/Hello.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { css } from '@emotion/css';
+import { useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { testIds } from '../components/testIds';
+import { PluginPage } from '@grafana/runtime';
+
+export function Hello() {
+  const s = useStyles2(getStyles);
+
+  return (
+    <PluginPage>
+      <div data-testid={testIds.hello.container}>
+        <h2>&#x1F44B; Welcome!</h2>
+        <p>
+          This mock plugin uses <span className={s.blue}>Role-Based Access Control (RBAC)</span> to control access to
+          the following pages:
+          <ul className={s.list}>
+            <li>
+              &#128196; Research Documents: Accessible to individuals with the{' '}
+              <span className={s.blue}>grafana-appwithrbac-app.papers:read</span> permission, which is granted by
```

Comment:
Nit: all these `s.blue` looks like links, can you use a different color or style?